frontend/src/utils/cabins.ts

```diff
@@ -129,3 +131,20 @@ export const generateEmailAndBookingInput: (
     checkOut: datePick.checkOutDate,
   };
 };
+
+/* 
+  Checks if a date is within the fall or spring booking semester.
+*/
+export const dateInBookingSemester = (date: dayjs.Dayjs, bookingSemester: BookingSemester): boolean => {
+  const inFallSemester =
+    (date.isAfter(bookingSemester.fallStartDate) || date.isSame(bookingSemester.fallStartDate), "day") &&
+    (date.isBefore(bookingSemester.fallEndDate) || date.isSame(bookingSemester.fallEndDate, "day"));
+
+  const inSpringSemester =
+    (date.isAfter(bookingSemester.springStartDate) || date.isSame(bookingSemester.springStartDate)) &&
+    (date.isBefore(bookingSemester.springEndDate) || date.isSame(bookingSemester.springEndDate));
```

Comment:
[`isSameOrAfter` and `isSameOrBefore`](https://day.js.org/docs/en/query/is-same-or-after) already exist.

Reply:
A design concern here is that this would require Hyttestyret (Hytteturstyret (?)) to update the dates once a year.